xla/pjrt/BUILD

```diff
@@ -362,7 +362,7 @@ cc_library(
     hdrs = ["metrics.h"],
     deps = [
         "//xla/stream_executor",
-        "//xla/stream_executor/gpu:gpu_init",
+        "//xla/stream_executor/gpu:gpu_init_impl",
```

Comment:
Should this change be here?

Reply:
Yes, otherwise metrics.cc will have undefined reference to 'stream_executor::GPUMachineManager(). As I mentioned we can remove this `if_static` in https://github.com/openxla/xla/blob/main/xla/stream_executor/gpu/BUILD#L184 to avoid the change on metrics.h deps, but not sure whether it's ok to do this way.